### ISSUE TITLE
EZ-logout

### ISSUE THREAD

**Issue by alinaivanovaoff** [rusq/slackdump#320]
Hi everyone! How can I perform an EZ-login reset? It seems that the "./slackdump -auth-reset" command isn't working.

```
./slackdump -auth-reset

flag provided but not defined: -auth-reset
Slackdump is a tool for exporting Slack conversations, emojis, users, etc.

This program comes with ABSOLUTELY NO WARRANTY; This is free software, and you
are welcome to redistribute it under certain conditions. Read LICENSE for more
information.

Usage:

  slackdump <command> [arguments]

The commands are:

  wiz         Slackdump Wizard
  export      exports the Slack Workspace or individual conversations
  dump        dump individual conversations or threads
  archive     archive the workspace or individual conversations on disk
  search      dump search results
  convert     convert slackdump chunks to various formats
  list        list users or channels
  emoji       download workspace emojis
  workspace   add or choose already existing workspace to run on
  tools       diagnostic tools
  config      API configuration
  format      converts the slackdump files to a human readable format
  view        View the slackdump files
  version     print version and exit

Use "slackdump help <command>" for more information about a command.

Additional help topics:

  whatsnew        what's new in this version
  login           login related information
  chunk           chunk file format specification

Use "slackdump help <topic>" for more information about that topic.
```

**Comment by rusq** [comment]
Hey @alinaivanovaoff ,

TL;DR, to EZ-logoff: `./slackdump workspace delete <name>`, i.e. `./slackdump workspace delete default`.

Long answer:

First off, let's be precise — it's not just some random EZ-login, it's EZ-Login **3000** 😂

Jokes aside, the `-auth-reset` moved in v3 to "workspace" group of commands.
To list all authenticated workspaces, one can use "list"
```
./slackdump workspace list
```

And to delete (equivalent of `-auth-reset`):
```
./slackdump workspace del <name>
```
where <name> is the name of the workspace, most likely it is "default" if it was slackdump authenticated as a side effect of running another command.

Key new feature, is that one can have multiple workspaces in v3, and "workspace select" allows to switch between them if needed.
